source/connect/in-use-encryption.txt

```diff
@@ -0,0 +1,263 @@
+.. _in-use-encryption-tab:
+
+================================
+In-Use Encryption Connection Tab
+================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+:guilabel:`In-Use Encryption` is an Enterprise/Atlas only feature. You need a replica set 
+to use this connection option. Your replica set can be a single node or larger. 
```

Comment:
```suggestion
or sharded cluster to use this connection option. Your replica set can be a single node or larger. 
```

Just a thought, but otherwise this might sound as if it only works when you’re interacting with a single replica set.

Reply:
Done.